OnlineShop/OnlineShop/Models/ScaffDir/User.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace online_s.ScaffDir;
+
+public partial class User
```

Comment:
why is partial?

Reply:
Classes were partial after scaffolding. I didn't noticed it's become redundant.